### ISSUE TITLE
Retest Only: Add/remove data elements from cdd data table - Intial drop down 508 issue

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#3402]
## Context 

Testing complete for 2835....... none of the new functionality is keyboard accessible. This will need to be added then reviewed again fully for 508 compliance. Unable to do any screen reader testing until keyboard accessibility is implemented.

**Comment by aprematta** [comment]
Testing complete, keyboard functionality has been added but there are additional 508 issues:
1. I can tab to the arrows used to sort the columns, but when I hit enter, nothing happens, the columns do not adjust to sort by ascending or descending.
2. I can tab to the arrows used to sort the columns, but when I hit enter focus is lost, it should stay on the arrow that I activated
3. The sorting arrows have aria-hidden=true which hides them to a screen reader
4. The sorting arrows lack a role of button
5. The sorting arrows lack a programatic name, suggest adding an aria label that says to sort by ascending/descending and the column name
6. When activating the additional options (with the 3 dots), the overlay with the 4 options display, but I can tab past them to other fields. The focus should be trapped until I make a selection or I hit the escape key
7. When I do make a selection, focus is lost, it should be placed back on the 3 dots
8. The 3 dots have aria-hidden=true which hides them to a screen reader
9. The 3 dots lack a role of button
10. The 3 dots lack a programatic name, suggest adding an aria label that says additional options and the column name
11. After making the customize column selection, The focus should be trapped until I make a selection or I hit the escape key
12. After making the customize column selection, I can tab to the select all and deselect all links, but when I hit enter, nothing happens.
13. Select all and deselect all links need to be underlined
14. The column title text field lacks a programatic label. Suggest turning the 'Find Column' div into a label tag and associating it to the input.
15. Focus is lost when I hit the Apply button.
